src/modules/Dropdown/Dropdown.js

```diff
@@ -39,7 +124,7 @@ export default class Dropdown extends Component {
 
   render() {
     const options = _.map(this.props.options, (opt, i) => {
-      return <option key={i} value={opt.value}>{opt.text}</option>;
+      return <option key={opt.value} value={opt.value}>{opt.text}</option>;
```

Comment:
Only warns if key is missing, there is no way for React to know if it is an index or a real key.